canonical/vanilla-framework
# Update Twitter/X icon and add support for dark theme to GitHub icon

## Done

Updates Twitter icon to use the new X logo. Adds dark background support to it.
As a drive-by adds dark background support for GitHub icon as well.

Fixes [WD-8070](https://warthogs.atlassian.net/browse/WD-8070)

## QA

- Open [demo](https://vanilla-framework-4978.demos.haus/docs/examples/patterns/icons/icons-social)
- Check if twitter logo is replaced with X
- Make sure both X and GitHub logos have white versions (to go on dark backgrounds)


## Screenshots

<img width="513" alt="image" src="https://github.com/canonical/vanilla-framework/assets/83575/f391c9ab-199c-4a25-9e84-39d28fa65b27">



[WD-8070]: https://warthogs.atlassian.net/browse/WD-8070?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
